src/user/dashboard/Community/community.scss

```diff
@@ -9,9 +9,62 @@
     bottom: 0px;
     left: 0px;
     overflow-y: scroll;
+  }*/
+  .sidebar {
+    background: #f5f5f5;
+    flex-grow: 1;
+    border-right: solid 1px #dfe9f1;
+    position: fixed;
+    top: 0px;
+    bottom: 0px;
+    left: 0px;
+    overflow-y: scroll;
+    transition: 0.3s ease-in-out;
+    width: 0;
+    overflow-x: hidden;
+  }
+  .sidebar-open {
+    width: 13vw;
+    background: #f5f5f5;
+    flex-grow: 1;
+    border-right: solid 1px #dfe9f1;
+    position: fixed;
+    top: 0px;
+    bottom: 0px;
+    left: 0px;
+    overflow-y: scroll;
+    overflow-x: hidden;
+    transition: 0.3s ease-in-out;
+  }
+
+  .sidebar-toggle {
+    position: sticky;
+    top: 30px;
+    left: 0;
+    bottom: 0;
+    transition: 0.3s ease-in-out;
+    width: 2.5rem;
+    height: 2.5rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    align-items: center;
+    background: #007bff;
```

Comment:
Can you implement this scss part as a single class and use it all the required pages instead of repeating it everywhere? (Have a look at the styling of modals).

Reply:
OK Done